recipes/photutils/meta.yaml

```diff
@@ -20,12 +20,13 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.9.*  # [not py36]
```

Comment:
Sorry. I hit send before finishing my comment. I also added np19. You can use that for all Python's too.